research/object_detection/meta_architectures/faster_rcnn_meta_arch_test.py

```diff
@@ -15,14 +15,16 @@
 
 """Tests for object_detection.meta_architectures.faster_rcnn_meta_arch."""
 
+from absl.testing import parameterized
```

Comment:
This looks like a new dependency. Should we update the installation docs?

Reply:
Actually I'm able to run this test externally without installing anything else, so I guess it should be fine.